app/services/republish_manual_service.rb

```diff
@@ -1,7 +1,7 @@
 class RepublishManualService
-  def initialize(published_listeners: [], draft_listeners: [], manual_id:)
-    @published_listeners = published_listeners
-    @draft_listeners = draft_listeners
+  def initialize(manual_id:)
+    @published_listeners = ManualObserversRegistry.new.republication
+    @draft_listeners = ManualObserversRegistry.new.update
```

Comment:
Minor: One difference is that we're now instantiating `ManualObserversRegistry` twice. I'm guessing there's no significant cost to doing this, but it might be worth noting in the commit note. Alternatively the instance could be extracted into a local variable in the constructor.